package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "desafio-front-taqtile",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@apollo/client": "3.7.17",
+    "@fortawesome/fontawesome-svg-core": "6.4.0",
+    "@fortawesome/free-brands-svg-icons": "6.4.0",
+    "@fortawesome/free-regular-svg-icons": "6.4.0",
+    "@fortawesome/free-solid-svg-icons": "6.4.0",
+    "@fortawesome/react-fontawesome": "0.2.0",
```

Comment:
Precisa dessas logo de cara?
Dá pra adicionar quando for necessário usar (se for necessário)